app/models/amends/AmendNewAnswerType.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.amends
+
+import play.api.libs.json.{Json, OFormat}
+
+
+sealed trait  AmendNewAnswerType{
+  val value: Option[String]
+  val hiddenMessage: String
+}
+case class AnswerWithValue(value: Option[String]) extends AmendNewAnswerType {
```

Comment:
is value optional? AnswerWithValue implies its not? 😕

Reply:
Yes, value could be null if we are not in amending mode